src/transformers/generation_tf_utils.py

```diff
@@ -678,6 +682,7 @@ def generate(
                 attention_mask=attention_mask,
                 output_attentions=output_attentions,
                 output_hidden_states=output_hidden_states,
+                return_dict=return_dict_in_generate,
```

Comment:
Without this line, the code block after it will throw exceptions when `return_dict_in_generate` is `True`

```
            if return_dict_in_generate:
                if output_attentions:
                    model_kwargs["encoder_attentions"] = encoder_outputs.attentions
                if output_hidden_states:
                    model_kwargs["encoder_hidden_states"] = encoder_outputs.hidden_states
```